JSDemos/Demos/VectorMap/CustomAnnotations/description.md

```diff
@@ -0,0 +1,9 @@
+Annotations are images and text blocks that provide additional information about the visualized data.
+
+To configure individual annotations, assign an array of objects to the [annotations](/Documentation/ApiReference/Data_Visualization_Widgets/dxVectorMap/Configuration/annotations/) option. To specify settings for all annotations, use the [commonAnnotationSettings](/Documentation/ApiReference/Data_Visualization_Widgets/dxVectorMap/Configuration/commonAnnotationSettings/) object. Individual settings take precedence over common settings.
```

Comment:
The first sentence first talks about "individual" annotations, then switches to an array (so, multiple objects). That's a bit confusing... 

```suggestion
To add annotations to a chart, assign an array of objects to the [annotations](/Documentation/ApiReference/Data_Visualization_Widgets/dxVectorMap/Configuration/annotations/) option. To specify settings for all annotations, use the [commonAnnotationSettings](/Documentation/ApiReference/Data_Visualization_Widgets/dxVectorMap/Configuration/commonAnnotationSettings/) object. Individual settings take precedence over common settings.
```

Reply:
I applied your suggestions, but let me clarify what I meant.

Each object in the annotations[] array represents an individual annotations, so users have to specif individual settings in these objects.